add localstack desktop app blog (#149)
localstack/localstack.github.io: content/blog/2023-10-23-introducing-localstack-desktop-application-for-local-cloud-development-testing/index.md

{{< img-simple src="localstack-desktop-resource-browser.png" width=300 alt="Image of LocalStack Desktop Application showcasing the local AWS Resource Browsers">}}

## What's new?

Last year, we released [LocalStack Cockpit](https://localstack.cloud/blog/2022-02-07-localstack-cockpit/) — Our first iteration towards building a fully integrated desktop client to control, access, and manage LocalStack with a click of a button. We received a [lot of feedback](https://github.com/localstack/cockpit/issues) from the community during the beta phase, and after spending months working on it, we formulated a new product for our users to use LocalStack more efficiently. 

To provide our users with the very best, we've made the decision to deprecate LocalStack Cockpit for new users. Going forward, we will help migrate our existing users and get our new users started with to the LocalStack Desktop Application.

{{< img-simple src="localstack-desktop-containers.png" width=300 alt="Image of LocalStack Desktop Application showcasing the running containers">}}

> The new LocalStack Desktop Application features an improved user experience which is in line with our Web Application experience, with [local Resource Browsers](https://docs.localstack.cloud/user-guide/web-application/resource-browser/) available for 30+ AWS services, and an intuitive way to manage your LocalStack container and view LocalStack logs through an easy-to-use interface. 

Reviewer: We also have a built-in CLI, which can be used to interact with the container - maybe something worth mentioning and including the screenshot of the logs?

> The LocalStack Desktop Application is available to download via an [executable from the Web Application]() or on the [Apple Store]() and [Windows Store](). You can download the application for free and get started!

Reviewer 2: Actually, we have not planned to release it to the Apple store (yet), so I would remove it...also you can already put links (but wait before merging this) since on the web application it will live at https://app.localstack.cloud/download and for the windows store the link is https://apps.microsoft.com/detail/9NTRNFT9ZWS2

lead: Get started with the new Desktop Application to seamlessly manage your LocalStack container while developing, testing, and debugging your cloud & serverless applications.
date: 2023-10-23
lastmod: 2023-10-23
images: []
contributors: ["LocalStack Team"]
tags: ['news']
show_cta_1: true
leadimage: 'introducing-localstack-desktop.png'
---

{{< img-simple src="introducing-localstack-desktop.png" >}}

We are excited to announce that we have released an all-new Desktop Application to make cloud development with LocalStack a breeze! The Desktop application underscores our unwavering commitment to make LocalStack accessible for our users, who would like to use a local cloud sandbox for development, testing, and experimentation, and truly embrace the inner dev loop. The Desktop Application now enables users to control, manage, and visualize over 80+ emulated AWS cloud services, running entirely on your local machine.

{{< img-simple src="localstack-desktop-resource-browser.png" width=300 alt="Image of LocalStack Desktop Application showcasing the local AWS Resource Browsers">}}

## What's new?

> Last year, we released [LocalStack Cockpit](https://localstack.cloud/blog/2022-02-07-localstack-cockpit/) — Our first iteration towards building a fully integrated desktop client to control, access, and manage LocalStack with a click of a button. We received a [lot of feedback](https://github.com/localstack/cockpit/issues) from the community during the beta phase, and after spending months working on it, we formulated a new product for our users to use LocalStack more efficiently. 

Reviewer 3: ```suggestion
Last year, we released [LocalStack Cockpit](https://localstack.cloud/blog/2022-02-07-localstack-cockpit/) — our first iteration towards building a fully integrated desktop client to control, access, and manage LocalStack with a click of a button. We received a [lot of feedback](https://github.com/localstack/cockpit/issues) from the community during the beta phase, and after spending months working on it, we formulated a new product for our users to use LocalStack more efficiently. 
```

> To provide our users with the very best, we've made the decision to deprecate LocalStack Cockpit for new users. Going forward, we will help migrate our existing users and get our new users started with to the LocalStack Desktop Application.

Reviewer 3: ```suggestion
To further streamline our product offering, we've made the decision to deprecate LocalStack Cockpit for new users. Going forward, we will help migrate our existing users and get our new users started with to the LocalStack Desktop Application.
```

The LocalStack Desktop Application is available to download via an [executable from the Web Application]() or on the [Apple Store]() and [Windows Store](). You can download the application for free and get started!

{{< img-simple src="localstack-desktop-logs.png" width=300 alt="Image of LocalStack Desktop Application showcasing the LocalStack logs">}}

## What's next?

For us, this is just the beginning. Stay tuned as we deliver many planned improvements over the next few weeks to make it even easier to build, test, and debug cloud & serverless applications. We have received great feedback from some of our existing users which is a testament to our commitment towards our community and users. 

It also sets the stage for us as we approach the next major release for LocalStack, where we aim to make our cloud emulator tooling accessible and open up our cloud development platform to the world.

> Share your feedback with us on our [Discuss Forum](https://discuss.localstack.cloud) so we can continue improving our local cloud developer experience! You can explore [new features on our documentation]() or request a demo to understand how to deploy your AWS infrastructure 10x faster locally and on CI with LocalStack!

Reviewer 3: Nit: I'd try to avoid using too many exclamation marks - typically maximum 1 per paragraph is a good guideline: 
```suggestion
Share your feedback with us on our [Discuss Forum](https://discuss.localstack.cloud) so we can continue improving our local cloud developer experience. You can explore [new features on our documentation]() or request a demo to understand how to deploy your AWS infrastructure 10x faster locally and on CI with LocalStack!
```
